project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clojurewerkz/elastisch "3.0.0-beta2-SNAPSHOT"
+(defproject clojurewerkz/elastisch "5.0.0-alpha1-SNAPSHOT"
```

Comment:
I'm not against jumping to `5.0` after `3.0` but I'm not going to accept a PR that changes project version. This is pull request etiquette 101.